frontend/app/src/utils/text.ts

```diff
@@ -21,3 +21,104 @@ export const toCapitalCase = (string: string): string => {
     return txt.charAt(0).toUpperCase() + txt.slice(1);
   });
 };
+
+/**
+ * Returns the plural of an English word.
+ *
+ * @export
+ * @param {string} word
+ * @param {number} [amount]
+ * @returns {string}
+ */
+export function pluralize(word: string, amount?: number): string {
```

Comment:
is this required for some protocol?

Reply:
This was asked by Left to pluralize the type 
![image](https://user-images.githubusercontent.com/26648140/163803227-1617ddbc-ef79-4db1-9ad1-ecbe6fef4aa9.png)

Reply:
Hmm but some don't have plural :D  like staking :wink: 

The rest look fine.

Reply:
Interesting, where I can see the list of the possible types? @LefterisJP I'll try to exclude them manually

Reply:
We may add more later but these are here: https://github.com/rotki/rotki/blob/435742c3bd305a6551250984c04ddf74a66009cd/rotkehlchen/accounting/mixins/event.py#L14-L24

I think only staking would need an exception

Reply:
Done ser 
![image](https://user-images.githubusercontent.com/26648140/163806142-6e15556d-9ca7-434c-8b06-a797939ceda3.png)

Reply:
Thanks